google/cloud/aiplatform/initializer.py

```diff
@@ -135,6 +137,53 @@ def get_client_options(
             api_endpoint=f"{region}-{prediction}{utils.PROD_API_ENDPOINT}"
         )
 
+    def get_resource_parent(
+        self, project: Optional[str] = None, location: Optional[str] = None
+    ) -> str:
+        """Get parent resource with optional project and location override.
+
+        Args:
+            project (str): GCP project. If not provided will use the current project.
+            location (str): Location. If not provided will use the current location.
+        Returns:
+            resource_parent: Formatted parent resource string.
+        """
+        return "/".join(
+            [
+                "projects",
+                project or self.project,
+                "locations",
+                location or self.location,
```

Comment:
The parent can be more complex than just `projects/*/locations/*`, e.g. model_evaluation's parents also include `models/*`.

Reply:
Renaming to common location path.

Reply:
Done.